test/over_react/component_declaration/builder_integration_tests/new_boilerplate/function_component_test.dart

```diff
@@ -134,6 +135,36 @@ main() {
       });
     });
   });
+
+  group('GenericUiProps', () {
+    late GenericUiProps props;
+
+    setUp(() {
+      final genericFactory = uiFunction<UiProps>((_) {}, UiFactoryConfig());
+      final factoryProps = genericFactory();
+      expect(factoryProps, isA<GenericUiProps>(), reason: 'test setup check');
+      props = factoryProps as GenericUiProps;
+    });
+
+    group('has functional overrides to members that are typically generated', () {
+      // staticMeta currently isn't implemented; not sure if/when we want to support that.
```

Comment:
Does this need to be a TODO with a ticket?

Reply:
Yeah I was on the fence on whether this was something we event wanted to do, but I'll create a ticket

Reply:
Updated: https://github.com/Workiva/over_react/pull/869/commits/80a4d6729995edff2affd3f5ff040d2bf4359218